From 415e2eca350d854bc0f8ff5dbde9e215a038b7cf Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Wed, 13 Jul 2016 16:49:16 +0100
Subject: .gitconfig 000: [PATCH 004/191] git wanted this

 chris@inside:~/arcola/wiki$ g-push
 warning: push.default is unset; its implicit value has changed in
 Git 2.0 from 'matching' to 'simple'. To squelch this message
 and maintain the traditional behavior, use:

   git config --global push.default matching

 To squelch this message and adopt the new behavior now, use:

   git config --global push.default simple

 When push.default is set to 'matching', git will push local branches
 to the remote branches that already exist with the same name.

 Since Git 2.0, Git defaults to the more conservative 'simple'
 behavior, which only pushes the current branch to the corresponding
 remote branch that 'git pull' uses to update the current branch.

 See 'git help config' and search for 'push.default' for further information.
 (the 'simple' mode was introduced in Git 1.7.11. Use the similar mode
 'current' instead of 'simple' if you sometimes use older versions of Git)

Conflicts:
	.gitconfig
---
 .gitconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.gitconfig b/.gitconfig
--- a/.gitconfig
+++ b/.gitconfig
@@ -16,3 +16,7 @@
 
 #[gpg]
 #	program = gpg+scrypt
+
+#[push]
+#	default = simple
+#wow old git can*not* deal with this
